docs/application/web/api/5.0/ui_fw_api/Wearable_UIComponents/wearable_sectionchanger.htm

```diff
@@ -16,7 +16,7 @@ <h1>Section Changer</h1>
 It gets to have a virtue of needlessness of changing page and the existance of same header within multiple <span style="font-family: Courier New,Courier,monospace">&lt;section&gt;</span> elements.</p>
 
 <video autoplay muted loop controls>
-	<source src="../../images/SectionChanger.ogv">
+	<source src="https://developer.tizen.org/sites/default/files/documentation/sectiON-CHANGEr.mp4">
```

Comment:
check file name - it should be lower case everywhere

Reply:
I changed names to camelCase